README.md

```diff
@@ -1,2 +1,34 @@
-# mobile-version-skeleton
-Portfolio setup and mobile version skeleton
+# Hello Project
+> This is my First Project in Microverse
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you improve your project description. Your project description should describe what about the project and not about Microvere.